GeosUtil/pressure_mod.F90

```diff
@@ -84,6 +84,7 @@ MODULE PRESSURE_MOD
 #endif
 
   REAL(fp)              :: a132_loc(133), b132_loc(133)
+  REAL(fp)              :: a91_loc(92),   b91_loc(92)
```

Comment:
Should 91 levels be included in the standard model?

Reply:
Let's add this. We have started to experiment with 91 model levels, so having this synced with the main code will be good.